src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java

```diff
@@ -73,6 +82,12 @@ public void put(Collection<SinkRecord> records) {
     );
     try {
       writer.write(records);
+    } catch (SchemaMismatchException sme) {
+      if (reporter != null) {
+        unrollAndRetry(records);
```

Comment:
Is there a mechanism in error reporting framework to turn off the additional unrolling? If I'm on 2.6+ but have DLQ turned off, do we still end up unrolling and sending each request individually to the database again before failing?

Reply:
If the DLQ is turned off, it will appear in the connector as `null` and will not go through the unrolling process. Does that answer your question?